text/0000-create-repo-issues.md

```diff
@@ -0,0 +1,103 @@
+* FCP &rarr; Final Comment Period. This is step taken on a particular RFC when enough discussion of the tradeoffs have taken place and the team is in a position to make a decision. Before entering FCP, super majority of the team must sign off.
+* PR &rarr; (Github) Pull Request.
+
+# Motivation
+[motivation]: #motivation
+
+As the number of RFCs increase, it has become increasingly hard to ensure that a given RFC has been implemented, or will be implemented. This RFC proposes an amendment to the [RFC process](https://github.com/buildpacks/rfcs#rfc-process).
+
+The current RFC process [defines](https://github.com/buildpacks/rfcs#rfc-process) that:
+> the team member who merges the pull request should do the following ... Create a corresponding issue in the appropriate repo.
+
+In practice, that adds extra work for the team member who merges in the PR, and it may slip through the cracks occasionally. Additionally, the sub-team maintainers are often the best people to create the issue, given their insight into how it should be implemented.
+
+As an additional consideration, it is often hard to track work from the RFC proposal to the implementation, and difficult to ensure that it was completed. If we use [Github references](https://docs.github.com/en/github/writing-on-github/autolinked-references-and-urls#issues-and-pull-requests) to link the RFC PR to the issue on the appropriate repository, we can better track the work, and be notified when it is fully implemented.
+
+# What it is
+[what-it-is]: #what-it-is
+This defines a process for sub-team maintainers to open issues on the relevant repositories, after FCP for the RFC concludes, before it is merged in.
+
+# How it Works
+[how-it-works]: #how-it-works
+When a PR has been accepted (after FCP has concluded), the sub-team maintainers make issues (tagging the RFC PR) in the relevant buildpack repos, and labels the PR `issues-created`. At that point, the RFC PR may be merged in.
+
+Given an RFC PR #002, which discusses a change relevant to the `pack CLI` to add in a `pack my-bags` command. After discussion in Working Group meetings and votes from the Core Team, the RFC is placed in FCP for 7 days. At the end of those, the platform maintainers should open an issue in the `pack CLI`, which tags the RFC PR. For example,
+
+```
+### Description
+
+To enable easier traveling, we would like to add a `pack my-bags` command
+
+### Proposed solution
+This issue is a request to implement the proposed solution as detailed in #buildpacks/rfcs#002
+
+### Additional context
+- This feature should be documented as part of standard pack CLI commands documentation.
+```
+
+After creating the issue, the maintainer should add the label `issues-created`  to the PR. Upon seeing that issue created, a member of the core team will merge it in.
```

Comment:
As part of the merge process, should the core team member add references to the created issues in the `CNB Issue` field in the RFC meta section?

Maybe I am misunderstanding the original intention of this field (cc @hone)? In practice, we seem to leave it blank most of the time, but it could be powerful navigational tool.

Example: https://github.com/buildpacks/rfcs/blob/main/text/0044-pack-publish-buildpack.md#meta

Reply:
yeah, that's the intention so we can reference the RFC discussion after it's been merged.

Reply:
Great point, @ekcasey. I'll add that in.